src/index.ts

```diff
@@ -127,6 +127,20 @@ class LinkSDK {
     window.open(`https://${this.domains.myaccount}/${MY_ACCOUNT.PATHS.OAUTH}${params}`, newTab ? '_blank' : '_self');
   }
 
+  // Open My Account and logs you out from the current session
+  public logout({ newTab = false }: IMyAccountOptions = {}): void {
+    if (!this.isInitialized) {
+      throw new Error('SDK not initialized');
```

Comment:
This this track by sentry? if not (as is an sdk) what is the benefit of throwing an error?

Reply:
Part of the SDK, it will warn the user that the SDK needs to be initialized before calling any other method.